yahoo-finance/YahooFinance.py

```diff
@@ -0,0 +1,12 @@
+from yahoo_finance import Share
+
+def get_data(ticker, start_date, end_date):
+	"""Creates wrapper instance of yahoo_finance Share class.
+		   :param ticker: official ticker for the company
+		   :params startDate, endDate: Datetime.date objects, determines which days to get stock price, inclusive
```

Comment:
`start_date` and `end_date`, and `datetime.date`, not `startDate`, `endDate`, `Datetime.date`.